phel.html.markdown

```diff
@@ -0,0 +1,333 @@
+
+# The first call in a file should be ns, to set the namespace
+(ns learn-phel)
+
+# More basic examples:
+
+# str will create a string out of all its arguments
+(str "Hello" " " "World") # => "Hello World"
+
+# Math is straightforward
+(+ 1 1) # => 2
+(- 2 1) # => 1
+(* 1 2) # => 2
+(/ 2 1) # => 2
+
+# Equality is =
+(= 1 1) # => true
+(= 2 1) # => false
+
+# You need not for logic, too
+(not true) # => false
+
+# Nesting forms works as you expect
+(+ 1 (- 3 2)) # = 1 + (3 - 2) => 2
+
+# Phel inherits PHP under the hood, so it can use native PHP (functions and classes) without any additional cost 
+# by using the `php/` prefix to all PHP native functions.
+
+# Types
+#############
+
+# Booleans are similar as the native PHP ones
+
+nil
+true
+false 
+
+# Symbols are used to name functions and variables in Phel
+# For example: symbol, snake_case_symbol, my-module/my-function
+
+# Keywords are like symbols that begin with a colon character. However, they are used as constants rather than a name for something.
+
+:keyword
+:0x0x0x
+::
+
+# Numbers in Phel are equivalent to numbers in PHP
+
+1337 # integer
++1337 # positive integer
+-1337 # negative integer
+
+1.234 # float
++1.234 # positive float
+-1.234 # negative float
+1.2e3 # float
+7E-10 # float
+
+# Strings are surrounded by double quotes. They almost work the same as PHP double quoted strings.
+# A string can be written in multiple lines. The line break character is then ignored by the reader.
+
+"hello world"
+
+"this is\na\nstring"
+
+"this
+is
+a
+string."
+
+"use backslack to escape \" string"
+
+"the dollar must not be escaped: $ or $abc just works"
+
+
+# Collections & Sequences
+#############
+
+# Lists are linked-list data structures, while vectors are array-backed.
+(type '(1 2 3)) # :list
+(type [1 2 3])  # :vector
+
+# A list would be written as just (1 2 3), but we have to quote
+# it to stop the reader thinking it's a function.
+# Also, (list 1 2 3) is the same as '(1 2 3)
+
+# You can produce a (non-lazy) sequence between a range. 
+(range 1 10 2) # <- (range from to step)
+(take 4 (range 10))
+
+# Use cons to add an item to the beginning of a list
+(cons 4 '(1 2 3)) # => (4 1 2 3)
+
+# Use push to add, and put to replace an item in a vector 
+(push [1 2 3] 4)  # => (1 2 3 4)
+(put [1 2 3] 1 4) # => (1 4 3)
+
+# Use concat to add lists or vectors together
+(concat [1 2] '(3 4)) # => [1 2 3 4]
+
+# Use filter, map to interact with collections
+(map inc [1 2 3]) # => [2 3 4]
+(filter even? [1 2 3]) # => [2]
+
+# Use reduce to reduce them. The initial-value is mandatory
+(reduce + 0 [1 2 3 4])
+# => (+ (+ (+ 1 2) 3) 4)
+# => 10
+
+(reduce push [] '(3 2 1))
+# = (push (push (push [] 3) 2) 1)
+# => [3 2 1]
+
+# Functions
+#############
+
+# Use fn to create new functions. A function always returns its last statement.
+(fn [] "Hello World") # => :function
+
+# You need extra parens to call it
+((fn [] "Hello World")) # => "Hello World"
+
+# You can create a var using def
```

Comment:
Better: You can bind a value to a symbol using def for definition

PS: `var` is also used for variables